core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender43.java

```diff
@@ -53,14 +57,29 @@ static ApacheSender43 create() {
         final ApacheSender43 sender = new ApacheSender43();
         Thread initThread = new Thread(
                 new Runnable() {
-
                     @Override
                     public void run() {
                         PoolingHttpClientConnectionManager cm = new PoolingHttpClientConnectionManager();
+                        final String[] allowedProtocols = SSLOptionsUtil.getAllowedProtocols();
+                        if (allowedProtocols != null) {
+                            SSLContext sslContext;
+                            try {
+                                sslContext = SSLContexts.custom().useProtocol("TLS").build();
+                            } catch (NoSuchAlgorithmException | KeyManagementException e) {
+                                if (InternalLogger.INSTANCE.isWarnEnabled()) {
```

Comment:
I'm thinking error level, or maybe even fatal(?)

Reply:
If i'm using the default SSLContext, this no longer applies.